substrate/frame/staking/src/tests.rs

```diff
@@ -7995,7 +8038,7 @@ mod ledger_recovery {
 			assert_eq!(Balances::balance_locked(crate::STAKING_ID, &333), lock_333_before); // OK
 			assert_eq!(Bonded::<Test>::get(&333), Some(444)); // OK
 			assert!(Payee::<Test>::get(&333).is_some()); // OK
-											 // however, ledger associated with its controller was killed.
+												// however, ledger associated with its controller was killed.
```

Comment:
```suggestion

// however, ledger associated with its controller was killed.
```

Reply:
for some reason the fmt is not working here (noticed that elsewhere). Better to add a break line and it should be fine.